src/Components/TabbedCard/TabbedCard.jsx

```diff
@@ -0,0 +1,39 @@
+import { useRef, useState } from 'react';
+import { get } from 'lodash';
+import PropTypes from 'prop-types';
+import { Row } from 'Components/Layout';
+import NavTabs from '../NavTabs/NavTabs';
+
+const TabbedCard = ({ tabs }) => {
+  const tabRef = useRef();
+
+  const [activeTab, setActiveTab] = useState(get(tabs, '[0].value') || '');
```

Comment:
```suggestion
  const [activeTab, setActiveTab] = useState(tabs?.[0]?.value || '');
```
to get rid of Lodash

Reply:
Is this mainly to reduce the number of dependencies in this file or because there's a con to using lodash?

Reply:
Lodash is significantly slower than vanilla js, so in cases where the vanilla js equivalent is just as easy to implement, we go with it